atlasdb-client/src/main/java/com/palantir/atlasdb/schema/stream/StreamStoreDefinition.java

```diff
@@ -0,0 +1,54 @@
+/**
+ * Copyright 2015 Palantir Technologies
+ *
+ * Licensed under the BSD-3 License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://opensource.org/licenses/BSD-3-Clause
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.palantir.atlasdb.schema.stream;
+
+import java.util.Map;
+
+import com.palantir.atlasdb.protos.generated.TableMetadataPersistence.ExpirationStrategy;
+import com.palantir.atlasdb.table.description.TableDefinition;
+import com.palantir.atlasdb.table.description.ValueType;
+import com.palantir.atlasdb.table.description.render.Renderers;
+import com.palantir.atlasdb.table.description.render.StreamStoreRenderer;
+
+public class StreamStoreDefinition {
+    final Map<String, TableDefinition> streamStoreTables;
+    String shortName, longName;
+    ValueType idType;
+    int inMemoryThreshold;
+    ExpirationStrategy expirationStrategy;
+
+    StreamStoreDefinition(Map<String, TableDefinition> streamStoreTables, String shortName, String longName, ValueType idType, int inMemoryThreshold, ExpirationStrategy expirationStrategy) {
+        this.streamStoreTables = streamStoreTables;
+        this.shortName = shortName;
+        this.longName = longName;
+        this.idType = idType;
+        this.inMemoryThreshold = inMemoryThreshold;
+        this.expirationStrategy = expirationStrategy;
+    }
+
+    public Map<String, TableDefinition> getTables() {
+        return streamStoreTables;
+    }
+
+    public StreamStoreRenderer getRenderer(String packageName, String name) {
+        return new StreamStoreRenderer(Renderers.CamelCase(longName), idType, packageName, name, inMemoryThreshold, expirationStrategy);
+    }
+
+    public String shortName() {
```

Comment:
probably going to swap this out to be getCleanupTasks, just getting lazy with how much work the refactor has been